.github/workflows/backend_tests.yml

```diff
@@ -48,6 +48,7 @@ jobs:
           BASE_REF: ${{ github.base_ref }}
           SKIP_SONAR: "true" # TODO remove when public
         run: |
+          ./mvnw clean install -DskipTests -Pprod
```

Comment:
we shouldn't build prod profile unless necessary (extra time building the frontend). As I'm fixing the startup issue (#758) let's revert this please